meson.build

```diff
@@ -118,6 +121,10 @@ if platform == 'windows'
   dxvk_name_prefix = ''
   compiler_args += ['-DDXVK_WSI_WIN32']
 else
+  dxvk_abi_version = '0'
+  dxvk_version = meson.project_version().strip('v').split('.')
+  dxvk_so_version = {'version': dxvk_abi_version + '.' + dxvk_version[0] + '.' + dxvk_version[1]}
```

Comment:
probably wants `+ '.' + dxvk_version[2]` as well?

Reply:
Alternatively make it `.0.231`

Reply:
Sure.

Reply:
I still think it would be nice to know the full version. We don't really do versioning other than "feels" in DXVK.

So let's have the third digit.

Reply:
Added in the latest revision!